### PR TITLE
feat(Look and feel): Re-write look and feel reference

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [RUNTIME-528](https://bonitasoft.atlassian.net/browse/RUNTIME-528)

### CODE REVIEW

File: modules/ROOT/pages/creating-a-new-look-feel.adoc [hunk 1]
```diff
@@ -1,181 +1,10 @@
...
-.page_performTask .body, .page_StartProcess .body, .page_DisplayCaseForm .body {
-  padding-bottom: 0 !important;
-  margin-bottom: 0 !important;
-  display: block !important;
-  overflow: hidden;
-}
-----
-
-With this definition, the form footer is displayed instead of the Portal footer, and the form's iframe is now contained in a table row. This is recommended for easier maintenance and to avoid issues at future migrations.
+The two places that still use the old Look & Feel are the wrapped PortalJs pages and the Bonita Login page. +
```
Comment (educhastenier) [inline modules/ROOT/pages/creating-a-new-look-feel.adoc]: For the lambda reader of the doc, `PortalJs` does not means anything, I suggest to NOT mention it.
Just precise the list of pages that use the old Look & Feel, that would be enough

File: modules/ROOT/pages/creating-a-new-look-feel.adoc [hunk 2]
```diff
@@ -1,181 +1,10 @@
...
+PortalJs pages take into account the theme of your application. If you want to modify their style, we strongly suggest xref:theme-development.adoc[you change the theme]. +
+For the Login page, you will still need to change the old Look & Feel in the above-mentioned folder. A solution is in the works to take into account the theme of the application.
```
Comment (educhastenier) [inline modules/ROOT/pages/creating-a-new-look-feel.adoc:10]: > A solution is in the works

What does this mean?

Comment (DumitruCorini) [reply modules/ROOT/pages/creating-a-new-look-feel.adoc:10]: https://www.google.com/search?q=+in+the+works

File: modules/ROOT/pages/creating-a-new-look-feel.adoc (hunk 1 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/creating-a-new-look-feel.adoc]: So, just "are the wrapped pages and the Bonita Login page"?

Comment (baptistemesta) [inline modules/ROOT/pages/creating-a-new-look-feel.adoc]:
```suggestion
The two places that still use the old Look & Feel are old pages not made using the UI Designer and the Bonita Login page. +
```

File: modules/ROOT/pages/creating-a-new-look-feel.adoc [hunk 3]
```diff
@@ -1,181 +1,10 @@
...
-  padding-bottom: 0 !important;
-  margin-bottom: 0 !important;
-  display: block !important;
-  overflow: hidden;
-}
-----
-
-With this definition, the form footer is displayed instead of the Portal footer, and the form's iframe is now contained in a table row. This is recommended for easier maintenance and to avoid issues at future migrations.
+The two places that still use the old Look & Feel are the wrapped PortalJs pages and the Bonita Login page. +
+PortalJs pages take into account the theme of your application. If you want to modify their style, we strongly suggest xref:theme-development.adoc[you change the theme]. +
```
Comment (baptistemesta) [inline modules/ROOT/pages/creating-a-new-look-feel.adoc]:
```suggestion
Those old pages take into account the theme of your application. If you want to modify their style, we strongly suggest xref:theme-development.adoc[you change the theme]. +
```

**Pull request merged**